docs/kratos/social-signin/65_netid.mdx

```diff
@@ -4,12 +4,71 @@ title: Add NetID as a social sign-in provider in Ory
 sidebar_label: NetID
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # NetID
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add NetID as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the NetID logo to start the configuration.
+3. Copy the **Redirect URI** and save it for later use.
+4. Create a [NetID Service](https://developerzone.netid.dev/latest/devportal/tutorial/services/) and
+   [NetID Client](https://developerzone.netid.dev/latest/devportal/tutorial/clients/).
+5. While creating the NetID Client, enter the saved **Redirect URI** from Ory to the **Callback URL** field of the NetID Client.
+6. After creating the NetID Client, the client is initially in sandbox mode, where only specified test user email addresses can use OAuth login.
+To enable an email address for OAuth, go to the NetID Service page and click **Add test user**.
```

Comment:
not clear

Reply:
Refactored the text to explain it a bit more:

6. After creating the NetID Client, the client is initially in _sandbox mode_, which is a pre-production state
   prior to the release of your NetID Service.
   If you want to test your NetID Client while it is in sandbox mode, you need to create one or more test users,
   by going to the NetID Service page and clicking **Add test user**.

Reply:
To much "if's" here. Instead of describing this order of things it'd be better to add this information to an admonition and say that the client is first in sandbox, then gets promoted to production. 

Also, this information should be presented in the CLI-focused flow too, shouldn't it? If I go straight to the CLI tab I won't know this. 

Formatting: don't use italics, please. See the guidelines for formatting.